custom_components/sony/media_player.py

```diff
@@ -297,10 +305,14 @@ def media_stop(self):
     def volume_up(self):
         """Send stop command."""
         self.sonydevice.volume_up()
+        time.sleep(0.5)
```

Comment:
Is the sleep really necessary?

Reply:
I couldn't find another solution, because when called without sleep, it will report old value